estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,32 @@
+from odoo import fields, models, api
+from dateutil.relativedelta import relativedelta
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate Property Offer"
+
+    price = fields.Float("Price")
+    status = fields.Selection(
+        string="Status",
+        selection=[("accepted", "Accepted"), ("refused", "Refused")],
+        help="Status is used to define the status",
+        copy=False)
+
+    partner_id = fields.Many2one("res.partner", string="Partner", required=True)
+    property_id = fields.Many2one("estate.property", string="Property ID", required=True)
```

Comment:
not really clear because it's not actually an ID: property_id is a record of the type property
property_id.id would be the Property ID